src/index.js

```diff
@@ -462,13 +520,13 @@ export const fuzzNewTargetExpression = f =>
   ap(Shift.NewTargetExpression, {}, f);
 
 export const fuzzObjectAssignmentTarget = f =>
-  ap(Shift.ObjectAssignmentTarget, {"properties": many(choose(fuzzAssignmentTargetPropertyIdentifier, fuzzAssignmentTargetPropertyProperty))}, f);
+  ap(Shift.ObjectAssignmentTarget, {"properties": many(choose(fuzzAssignmentTargetPropertyIdentifier, fuzzAssignmentTargetPropertyProperty)), "rest": opt(choose(fuzzAssignmentTargetIdentifier, choose(fuzzComputedMemberAssignmentTarget, fuzzStaticMemberAssignmentTarget)))}, f);
```

Comment:
```suggestion
  ap(Shift.ObjectAssignmentTarget, {"properties": many(choose(fuzzAssignmentTargetPropertyIdentifier, fuzzAssignmentTargetPropertyProperty)), "rest": opt(choose(fuzzAssignmentTargetIdentifier, fuzzComputedMemberAssignmentTarget, fuzzStaticMemberAssignmentTarget))}, f);
```

Reply:
No, having a 50% probability of being an identifier is intentional and matches the rest of this file.